samigo/samigo-app/src/webapp/jsf/delivery/tableOfContents.jsp

```diff
@@ -159,7 +159,7 @@ function saveTime()
 <h:inputHidden id="outoftime" value="#{delivery.timeOutSubmission}"/>
 <h:commandLink id="submitforgrade" action="#{delivery.submitForGrade}" value="" />
 
-    <h:messages styleClass="messageSamigo" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
+    <h:messages styleClass="sak-banner-warn" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
```

Comment:
You've used .sak-banner-error for others like this. Why did you use .sak-banner-warn here?

Reply:
I can make them all sak-banner-error. I was debating which one to make the messageSamigos and was switching between the two because I wasn't sure.

Reply:
Ya, we might have to do some further revisions in future PRs. I think that "messageSamigo" was used generically, so we might have to split up the method to be able to take different message types specific to the case. For now, let's keep them all consistent and then we can revise them later. Thanks for making this change.